### ISSUE TITLE
Fix iMessage Images Problems

### ISSUE THREAD

**Issue by eXhumer1149** [kpwn/yalu102#294]
No images in #image after jailbreak, happens after installing Substrate. Images appear if I remove Substrate.

**Comment by Crazy-Clown** [comment]
[https://github.com/kpwn/yalu102/issues/245](url)

**Comment by eXhumer1149** [comment]
404 Didn't find anything here.

**Comment by kpwn** [comment]
I wonder if this is a substrate bug or just some additional kernel patch is needed.

**Comment by eXhumer1149** [comment]
Nvm

**Comment by eXhumer1149** [comment]
Will I remove this KPWN?

**Comment by eXhumer1149** [comment]
Keep it closed or open it again?

**Comment by nullpixel1** [comment]
Open, happens to me without substrate

**Comment by ccsasuke** [comment]
I might be wrong, but I remember this was working on b1. And I just checked, it was indeed broken using b6.

**Comment by Aaahh** [comment]
Broken on beta 3 as well

**Comment by kpwn** [comment]
You should be able to install b1 on a device running b6 without issues.

**Comment by ccsasuke** [comment]
Yes, I'm planning to try that and confirm what I remembered was true or not tonight. Just saying that so that other people can try b1 to see if it's indeed the case.

**Comment by JMccormick264** [comment]
this is happening to me. I just updated to latest

**Comment by JMccormick264** [comment]
@kpwn I have a crashReport labeled "HashtagImagesExtension". Do you want the report? Crash logs are working now, thank you!

**Comment by kpwn** [comment]
I do!

**Comment by JMccormick264** [comment]
Here you go http://pastebin.com/8HEMitYt
I am on iOS 10.1.1 running latest 10.2 beta

**Comment by kpwn** [comment]
abort() called - syslog probably has more info

**Comment by JMccormick264** [comment]
syslog isn't clickable in crashReporter. where is it in the file system?

**Comment by kpwn** [comment]
you can use libimobiledevice's idevicesyslog with a computer.

**Comment by JMccormick264** [comment]
I will have to do that when I get home, I am not at a computer. No way to find it in iFile?

**Comment by msamuel002** [comment]
wdym by "substrate"? is there a beta substrate for ios 10 by saurik? or is he going to release that only once (stable)?

**Comment by msamuel002** [comment]
lol if i knew @kpwn address, i would send him like 100 boxes of Club Mate lol for all he has done for us

**Comment by tangalbert919** [comment]
Looks like I'm getting the same problem as well.
OH MY F***ING GOD MY DEVICE NEEDS TO STOP DOWNLOADING IOS 10.2.1! I'M NOT UPDATING, NO MATTER WHAT!

**Comment by JMccormick264** [comment]
I can't get the syslog. I do not have a Mac and the libmobiledevice isn't working on windows

**Comment by ccsasuke** [comment]
The syslog @kpwn : http://pastebin.com/mTLn8Uew

**Comment by Aaahh** [comment]
Switching games are effected too

**Comment by Aaahh** [comment]
Like the imessgaes games

**Comment by Aaahh** [comment]
Dictation is broken too, the voice to text

**Comment by astepanov96** [comment]
^ Dictation works for me on 6S 10.2 in multiple languages. #images is an open bug for everyone.

**Comment by Aaahh** [comment]
Dictation works now 
When it wasnt enabled the popup would just keep popping up after click enable

**Comment by justinoboyle** [comment]
Message apps seem to now work properly on iPhone 6.

**Comment by astepanov96** [comment]
^ Are you on another beta or something?

**Comment by mmhobi7** [comment]
Still broken here

**Comment by skimaskngun** [comment]
what do you guys mean when you say images are broke? in the message app? everything is fine on my 6s yalu beta7

**Comment by mmhobi7** [comment]
Messages
New message
Type in someone with I messages into number
Send something
Click the application button next to to the text box
Click on the apps button in the lower corner
Click images

**Comment by ReddestDream** [comment]
So, I've done some tests, and this really looks like a Substrate issue. I can't reproduce this issue in No Substrate mode, and I can clearly reproduce it once Substrate is loaded. I don't think it's a daemon issue. I think that Substrate just needs to not be in HashtagImageExtension . . . :/

Tested on iTouch 6G (iOS 10.2: Yalu b7 + the fud fix). Please see syslogs and symbolicated crashlogs from the recently updated CrashReporter:

Crash Log: http://pastebin.com/mKLaTtzc

Syslog: http://pastebin.com/UJQeja1T

**Comment by ccsasuke** [comment]
Can confirm. Issue not reproduced in No Substrate mode. Didn't realize this since I had tried Safe Mode before, in which the bug was indeed present.
